packages/schemas/src/gen/index.ts

```diff
@@ -70,7 +71,8 @@ const generate = async () => {
                 const isArray = Boolean(/\[.*]/.test(type)) || restLowercased.includes('array');
                 const hasDefaultValue = restLowercased.includes('default');
                 const nullable = !restLowercased.includes('not null');
-                const primitiveType = getType(type);
+                const primitiveType = getType(removeParentheses(type));
```

Comment:
will this case ever happen that you need to remove parentheses on `type`?

Reply:
Since we only need the type name when we need to get its corresponding primitive type, so remove the parentheses.